main.go

```diff
@@ -160,19 +173,13 @@ func buildProxy(host, bind *url.URL, tokenSource oauth2.TokenSource) *httputil.R
 			return
 		}
 
-		// Get the id_token from the oauth token.
-		idTokenRaw := token.Extra("id_token")
-		if idTokenRaw == nil {
+		// Get the id_token.
+		idToken := token.AccessToken
+		if idToken == "" {
 			*r = *r.WithContext(context.WithValue(ctx, contextKeyError,
 				fmt.Errorf("missing id_token")))
```

Comment:
Should this message be updated?

Reply:
What are you thinking it should say?

Reply:
Access token instead of id_token?

Reply:
It's still an ID Token, it just comes into the AccessToken field on the oauth2 struct.